### PR TITLE
OSDOCS-7723:Compliance Operator 1.3.0 Release Notes

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
Version(s): 4.11, 4.12, 4.13, 4.14  (branched from main, cherrypick to older releases)

Issue: https://issues.redhat.com/browse/OSDOCS-7723

Link to docs preview: https://64681--docspreview.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes

SME review: 

QE review:
- [x] QE has approved this change.

Additional information: 
This is an enhancement release, there are no bug fixes for this release.

### CODE REVIEW

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 1]
```diff
@@ -15,6 +15,28 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+== OpenShift Compliance Operator 1.3.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5102[RHBA-2023:5102 - OpenShift Compliance Operator enhancement update]
+
+[id="compliance-operator-1-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Compliance Operator 1.3.0 now supports creation of the DISA-STIG profile within the supported profiles module for {product-title} 4.14. See https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles
```
Comment (sheriff-rh) [inline security/compliance_operator/compliance-operator-release-notes.adoc:28]: We aren't "creating" the DISA-STIG profile, and the profile will be available on all supported branches (4.11+).  Additionally, 4.14 is not released content at this time so we shouldn't be mentioning it anywhere in our documentation. Try: 
"The DISA-STIG profile is now available for Compliance Operator 1.3.0. "

Additionally, you have a URL where there should be an xref.

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 2]
```diff
@@ -15,6 +15,28 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+ for more information.
+
+* Compliance Operator 1.3.0 now provide support to IBM Power and IBM Z for NIST moderate profiles.
```
Comment (sheriff-rh) [inline security/compliance_operator/compliance-operator-release-notes.adoc:31]: Couple things: we have IBM architecture attributes you can use. I also recommend using the full title for the profiles, of which there are two: platform and node.

"Compliance Operator 1.3.0 now supports :ibmpowerProductName: and :ibmzProductName: for NIST 800-53 Moderate-Impact Baseline for Red Hat OpenShift platform and node profiles. "

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 3]
```diff
@@ -15,6 +15,28 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+[id="compliance-operator-1-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Compliance Operator 1.3.0 now supports creation of the DISA-STIG profile within the supported profiles module for {product-title} 4.14. See https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles
+ for more information.
+
+* Compliance Operator 1.3.0 now provide support to IBM Power and IBM Z for NIST moderate profiles. 
++
+[IMPORTANT]
```
Comment (sheriff-rh) [inline security/compliance_operator/compliance-operator-release-notes.adoc:33]: This is probably okay to add, as users may still be upgrading from versions prior to 1.2.0. We should run this past engineering and QE to be sure. We may need to rewrite, something like "If you are upgrading from a version prior to CO 1.2.0, ..." etc.

**Review by sheriff-rh — CHANGES_REQUESTED** [review]
This is a start, but you're missing a couple things from the [JIRA](https://issues.redhat.com/browse/OSDOCS-7723): 

1. Add DISA-STIG to the [supported profiles module](https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles). You may need to work with engineering to get all of the information for the fields (title, profile title, is there one for platform and node, supported architectures, etc). 

2. Add IBM Power and Z support to the OCP4 NIST moderate profiles in the same table as above. That's  `ocp4-moderate` and `ocp4-moderate-node` need to be updated with `s390x`  and `ppc64le` architectures.

**Comment by sheriff-rh** [comment]
> Version(s): 4.14

This is 4.11+, all content related to the Compliance Operator applies to all branches.

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 1 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:28]: Fixed - Thank you!

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 2 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:31]: Fixed, thank you!

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 3 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:33]: I'll check that with eng and QE,

Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc]: I've deleted the IMPORTANT block per SME advisement. Thank you!

File: modules/compliance-supported-profiles.adoc [hunk 4]
```diff
@@ -64,13 +87,17 @@ The Compliance Operator provides the following compliance profiles:
 |0.1.39+
 |link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
 |`x86_64`
+ `ppc64le`
```
Comment (sheriff-rh) [inline modules/compliance-supported-profiles.adoc:90]: We don't need to add `s390x` or `ppc64le` support for `rhcos4-moderate`, just `ocp4-moderate` and `ocp4-moderate-node`.

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 5]
```diff
@@ -15,6 +15,21 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+== OpenShift Compliance Operator 1.3.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5102[RHBA-2023:5102 - OpenShift Compliance Operator enhancement update]
+
+[id="compliance-operator-1-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The DISA-STIG profile is now available for Compliance Operator 1.3.0. See xref:../../security/compliance_operator/compliance-operator-supported-profiles.adoc#compliance-supported-profiles_compliance-operator-supported-profiles[Supported compliance profiles]
```
Comment (sheriff-rh) [inline security/compliance_operator/compliance-operator-release-notes.adoc:28]: I would update "DISA-STIG profile" with the full name: 

"The Defense Information Systems Agency Security Technical Implementation Guide (DISA STIG) for {product-title} is now available..." 

Additionally, we need a period after the xref sentence. I would also add something like:

"For more information, see xref:..[words]." 

Or: 
"See xref:..[words] for additional information."

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 6]
```diff
@@ -15,6 +15,21 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+
+* Compliance Operator 1.3.0 now supports {ibmpowerProductName} and {ibmzproductname} for NIST 800-53 Moderate-Impact Baseline for {product-title} platform and node profiles.
```
Comment (sheriff-rh) [inline security/compliance_operator/compliance-operator-release-notes.adoc:30]: Two empty spaces at the end of this line.

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 7]
```diff
@@ -15,6 +15,21 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
...
+* link:https://access.redhat.com/errata/RHBA-2023:5102[RHBA-2023:5102 - OpenShift Compliance Operator enhancement update]
+
+[id="compliance-operator-1-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The DISA-STIG profile is now available for Compliance Operator 1.3.0. See xref:../../security/compliance_operator/compliance-operator-supported-profiles.adoc#compliance-supported-profiles_compliance-operator-supported-profiles[Supported compliance profiles]
+
+* Compliance Operator 1.3.0 now supports {ibmpowerProductName} and {ibmzproductname} for NIST 800-53 Moderate-Impact Baseline for {product-title} platform and node profiles.  
+
+
```
Comment (sheriff-rh) [inline security/compliance_operator/compliance-operator-release-notes.adoc:32]: Delete one of these extra lines.

**Review by sheriff-rh — COMMENTED** [review]
Looks good! Some small nits for your consideration.

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 7 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:32]: Done, thank you!

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 6 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:30]: Done, thank you!

File: security/compliance_operator/compliance-operator-release-notes.adoc (hunk 5 above)
Comment (GroceryBoyJr) [reply security/compliance_operator/compliance-operator-release-notes.adoc:28]: Done, thank you!

File: modules/compliance-supported-profiles.adoc (hunk 4 above)
Comment (GroceryBoyJr) [reply modules/compliance-supported-profiles.adoc:90]: Done! I don't know how you knew that, let's discuss later.

**Comment by GroceryBoyJr** [comment]
Request your review, @SNiemann15

**Comment by GroceryBoyJr** [comment]
Request your review, @rhmdnd

**Comment by GroceryBoyJr** [comment]
Request your review, @xiaojiey

**Review by rhmdnd — COMMENTED** [review]
/lgtm

**Comment by SNiemann15** [comment]
Looks good for IBM Z 

@juliemathew Can you please confirm for IBM Power

**Comment by juliemathew** [comment]
Looks good for IBM P.

**Comment by GroceryBoyJr** [comment]
> This is a start, but you're missing a couple things from the [JIRA](https://issues.redhat.com/browse/OSDOCS-7723):
> 
>     1. Add DISA-STIG to the [supported profiles module](https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles). You may need to work with engineering to get all of the information for the fields (title, profile title, is there one for platform and node, supported architectures, etc).
> 
>     2. Add IBM Power and Z support to the OCP4 NIST moderate profiles in the same table as above. That's  `ocp4-moderate` and `ocp4-moderate-node` need to be updated with `s390x`  and `ppc64le` architectures.

These items are done.

**Review by sheriff-rh — APPROVED** [review]
LGTM, adding peer review label done label. I will merge when the errata is live.

**Comment by sheriff-rh** [comment]
Errata is live; merging.

**Pull request merged**